src/web/core/src/main/java/org/geoserver/web/GeoServerBasePage.java

```diff
@@ -223,6 +224,10 @@ protected String load() {
 
     @Override
     public void renderHead(IHeaderResponse response) {
+
+        // includes jquery, rehired by the placeholder plugin
```

Comment:
oops you are correct ... I don't really know why and how I wrote "rehired" instead of "required"